test/frontend/EditorForm.test.js

```diff
@@ -0,0 +1,57 @@
+/* global describe it */
+import 'jsdom-global/register'
+import React from 'react'
+import Enzyme, { mount } from 'enzyme'
+import { expect } from 'chai'
+import 'regenerator-runtime/runtime'
+
+import AdminEditorForm from '../../src/Pages/Profile/admin/EditorForm'
+import Adapter from 'enzyme-adapter-react-16'
+import { Modal, Button, FormGroup } from 'reactstrap'
+
+Enzyme.configure({ adapter: new Adapter() })
+
+describe('<AdminEditorForm />', () => {
+  const membership = [
+    { value: 0, name: 'Keep Same' },
+    { value: 2, name: '2 semesters' }
+  ]
+
+  const formGroups = [
+    {
+      label: 'First Name',
+      placeholder: 'First',
+      handleChange: () => {}
+    },
+    {
+      label: 'Last Name',
+      placeholder: 'Last',
+      handleChange: () => {}
+    }
+  ]
+
+  const wrapper = mount(
+    <AdminEditorForm
+      formGroups={formGroups}
+      membership={membership}
+      setNumberOfSemestersToSignUpFor={() => {}}
+      setPagesPrinted={e => {}}
+      handleSubmissionToggle={() => {}}
+      handleToggle={() => {}}
+      setuserMembership={() => {}}
+      toggle
+    />
+  )
+
+  it('Should render a <Modal /> component with one child', () => {
+    expect(wrapper.find(Modal)).to.have.lengthOf(1)
+  })
+
+  it('Should render a <Button /> component with 4 children', () => {
+    expect(wrapper.find(Button)).to.have.lengthOf(4)
+  })
+
+  it('Should render a <FormGroup /> component with 8 children', () => {
+    expect(wrapper.find(FormGroup)).to.have.lengthOf(8)
```

Comment:
ensure that the [rendered labels](https://github.com/SCE-Development/Core-v4/blob/628b21b3ff637a44eb7fa568212690c8d40bd347/src/Pages/Profile/admin/EditorForm.js#L40) have the proper text values